### PR TITLE
Improve handling cloudflare errrors

### PR DESCRIPTION
**Pull request by cpaillet**
- If a domain is in a deletion process, we cannot get analytics.
      On this error, we continue on others domains, now

### CODE REVIEW

**Comment by mougams** [comment]
You should split the bump from the error handling in a separate PR

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def _read_reqs(relpath):
 
 setuptools.setup(
     name='cloudflare-exporter',
-    version='0.6',
+    version='0.7',
```
Comment (mougams) [inline setup.py:20]: Split in separate commit

**Review by mougams — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (cpaillet) [reply setup.py:20]: discuss offline, we will use https://docs.openstack.org/pbr/2.1.0/ for next release
